src/python/nimbusml.pyproj

```diff
@@ -30,69 +30,70 @@
     <Compile Include="docs\sphinx\conf.py" />
     <Compile Include="nimbusml\base_predictor.py" />
     <Compile Include="nimbusml\base_transform.py" />
-    <Compile Include="nimbusml\cluster\_kmeansplusplus.py" />
+    <Compile Include="nimbusml\cluster\kmeansplusplus.py" />
```

Comment:
Why were these renamed (see all changes in the file)? `kmeansplusplus.py` (and all the other files) without the underscore should not exist.